kylin-spark-project/kylin-spark-metadata/src/main/spark31/org/apache/kylin/engine/spark/cross/CrossDateTimeUtils.scala

```diff
@@ -44,7 +44,7 @@ object CrossDateTimeUtils {
   }
 
   def daysToMillis(days: Int): Long = {
-    DateTimeUtils.daysToMicros(days, DEFAULT_TZ_ID)
+    DateTimeUtils.daysToMicros(days, DEFAULT_TZ_ID) / 1000
```

Comment:
Using `DateTimeUtils.microsToMillis(DateTimeUtils.daysToMicros(days, DEFAULT_TZ_ID))`

Reply:
And in millisToDays function , using `DateTimeUtils.microsToDays(DateTimeUtils.millisToMicros(millis), DEFAULT_TZ_ID)`